types/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// TypeScript Version: 3.8
+
+import type {Plugin} from 'unified'
+
+/**
+ * Remark plugin to remove Markdown links, images, references, and definitions.
+ */
+declare const remarkUnlink: Plugin
```

Comment:
this plugin does not accept settings
```suggestion
declare const remarkUnlink: Plugin<[]>
```

Reply:
right! ty I'll squash this change

Reply:
actually with this I'll get

```
  types/index.d.ts:8:36
  ✖  8:36  Don't use [] as a type. Don't use the empty array type []. It only allows empty arrays. Use SomeType[] instead.  @typescript-eslint/ban-types
```

Reply:
From xo?
Yeah, xo is good at linting full TS files but not designed for dts files.
https://github.com/remcohaszing/mdast-util-definitions/blob/b8b1302c70e2387247b97d4469742608491a1237/package.json#L49
is an approach used elsewhere. DTSlint is better aligned with and designed for dts file checking.